### DISCUSSION TITLE
Customizing quploader to show only 1 uploaded file. How to?

### DISCUSSION THREAD

**Discussion by Claus1** [quasarframework/quasar#11103] [CLI - PWA mode] [answered]
My app uses quploder for testing images. When the user uploads an image, he expects to see only it as the current. But images are stacked below visible area and quploder continues to show non-actual images.
![image](https://user-images.githubusercontent.com/1247062/138748293-40d69837-4668-46c7-9c8e-107a6121cc61.png)
I need to replace the old image by new one and make it fully visible.  How to?

**Comment by MilosPaunovic** [comment]
Does `max-files` prop set to 1 solves your issue?
https://quasar.dev/vue-components/uploader#restricting-upload

**Reply by Claus1** [reply]
nope. it does not allow to upload new one with  max-files="1", if it's filled. it has to replace automatically, without extra movements.

**Comment by metalsadman** [comment, marked as answer]
@Claus1 check the [api](https://quasar.dev/vue-components/uploader#quploader-api), you can add/remove files from the list manually using `methods` all the while listening for `events`. example https://codepen.io/metalsadman/pen/VwzpNpm?editors=1010

**Comment by Claus1** [comment]
@metalsadman Thanks a lot!!
